.github/workflows/collector.yml

```
@@ -168,15 +175,19 @@ jobs:
     - build-collector-image-remote-vm
     name: Create Multiarch manifest
     runs-on: ubuntu-24.04
-    if: |
-      github.event_name != 'pull_request' ||
-      contains(github.event.pull_request.labels.*.name, 'run-multiarch-builds')
     env:
       ARCHS: amd64 ppc64le s390x arm64
 
     steps:
       - uses: actions/checkout@v4
 
+      - name: Set ARCHS
+        run: |
+          if [[ "${GITHUB_EVENT_NAME}" == "pull_request" && "${{ contains(github.event.pull_request.labels.*.name, 'run-multiarch-builds') }}" != "true" ]]; then
+            ARCHS="amd64 arm64"
+            echo "ARCHS=$ARCHS" >> "$GITHUB_ENV"
+          fi
```

This might be more readable.
```suggestion
      - name: Set ARCHS
        if: github.event_name == 'pull_request' && !contains(github.event.pull_request.labels.*.name, 'run-multiarch-builds'
        run: |
          ARCHS="amd64 arm64"
          echo "ARCHS=$ARCHS" >> "$GITHUB_ENV"
```